provisioners/configure.sh

```diff
@@ -21,6 +21,12 @@ echo "Add custom sources"
 cp $TEMPLATES_PATH/usr/share/keyrings/*.asc /usr/share/keyrings/
 cp $TEMPLATES_PATH/etc/apt/sources.list.d/*.sources /etc/apt/sources.list.d/
 
+# Set up the correct node source
+export NODE_VERSION=14
```

Comment:
When the time comes, hopefully this will make it more obvious where/how to upgrade from 14 to 16!